blocks/block-edit/context.js

```diff
@@ -4,7 +4,7 @@
 import { createContext, createHigherOrderComponent } from '@wordpress/element';
 
 const { Consumer, Provider } = createContext( {
-	isSelected: true,
+	isSelected: false,
```

Comment:
Should be `false` since we migrated all core blocks to work with the block edit context.